tests/array/sort.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable require-jsdoc */
+
+import utils from "../../dist/main";
+
+import assert from "assert";
+
+function arraySortTests(): void {

```

Comment:
Codacy has a fix for the issue: [Missing JSDoc comment.](https://app.codacy.com/gh/RusAnonym/rus-anonym-utils/pullRequest?prid=7806944)
```suggestion
/**
```